client/css/style.scss

```diff
@@ -103,4 +103,8 @@
   transition: opacity 400ms ease-in;
 }
 
-
+.group__table {
```

Comment:
In following with bulma could we try `has-margin-5` or something like that, and then just put it on the buttons themselves?